packages/buidler-core/sample-project/test/sample-test.js

```diff
@@ -1,34 +1,14 @@
-const Greeter = artifacts.require("Greeter");
+const { expect } = require("chai");
 
-// Traditional Truffle test
-contract("Greeter", accounts => {
+describe("Greeter", function() {
   it("Should return the new greeting once it's changed", async function() {
-    const greeter = await Greeter.new("Hello, world!");
-    assert.equal(await greeter.greet(), "Hello, world!");
+    const Greeter = await ethers.getContractFactory("Greeter");
+    const greeter = await Greeter.deploy("Hello, world!");
+    
+    await greeter.deployed();
+    expect(await greeter.greet()).to.equal("Hello, world!");
 
     await greeter.setGreeting("Hola, mundo!");
-
-    assert.equal(await greeter.greet(), "Hola, mundo!");
-  });
-});
-
-// Vanilla Mocha test. Increased compatibility with tools that integrate Mocha.
-describe("Greeter contract", function() {
-  let accounts;
-
-  before(async function() {
-    accounts = await web3.eth.getAccounts();
-  });
-
-  describe("Deployment", function() {
```

Comment:
why did you remove this test rather than rewrite with ethers?

Reply:
I rewrited the test using ethers but I removed one of them because it was using TruffleContract (`contract` instead of Mocha's `describe`). Also, both tests were doing the same and `accounts` wasn't being used.